internal/ingest/ingest.go

```diff
@@ -25,6 +26,7 @@ type Configs struct {
 	StartLedger          int
 	EndLedger            int
 	LogLevel             logrus.Level
+	AppTracker           sentry.SentryTracker
```

Comment:
Here you should use the interface!

```suggestion
	AppTracker           apptracker.AppTracker
```

Reply:
Done